pointcloud/basic_octree_utils_test.go

```diff
@@ -127,7 +128,8 @@ func TestSplitIntoOctants(t *testing.T) {
 
 		basicOct.node = newLeafNodeFilled(r3.Vector{X: 0, Y: 0, Z: 10}, NewValueData(1.0))
 		err1 := basicOct.Set(r3.Vector{X: 0, Y: 0, Z: 0}, NewValueData(1.0))
-		test.That(t, err1, test.ShouldBeError, errors.Errorf("error in splitting octree into new octants: %v", err))
```

Comment:
[nit] don't know why this is changing - seems less robust this way

Reply:
I'd prefer to keep it the old way as well if possible.

Reply:
changed.